detectron2/utils/testing.py

```diff
@@ -153,3 +160,321 @@ def reload_lazy_config(cfg):
         fname = os.path.join(d, "d2_cfg_test.yaml")
         LazyConfig.save(cfg, fname)
         return LazyConfig.load(fname)
+
+
+def min_torch_version(min_version: str) -> bool:
+    """
+    Returns True when torch's  version is at least `min_version`.
+    """
+    try:
+        import torch
+    except ImportError:
+        return False
+
+    installed_version = version.parse(torch.__version__.split("+")[0])
+    min_version = version.parse(min_version)
+    return installed_version >= min_version
+
+
+def register_custom_op_onnx_export(
+    opname: str, symbolic_fn: Callable, opset_version: int, min_version: str
+) -> None:
+    """
+    Register `symbolic_fn` as PyTorch's symbolic `opname`-`opset_version` for ONNX export.
+    The registration is performed only when current PyTorch's version is < `min_version.`
+    IMPORTANT: symbolic must be manually unregistered after the caller function returns
+    """
+    if min_torch_version(min_version):
+        print(
+            f"_register_custom_op_onnx_export({opname}, {opset_version}) will be skipped."
+            f" Installed PyTorch {torch.__version__} >= {min_version}."
+        )
+        return
+    register_custom_op_symbolic(opname, symbolic_fn, opset_version)
+    print(f"_register_custom_op_onnx_export({opname}, {opset_version}) succeeded.")
+
+
+def unregister_custom_op_onnx_export(opname: str, opset_version: int, min_version: str) -> None:
+    """
+    Unregister PyTorch's symbolic `opname`-`opset_version` for ONNX export.
+    The un-registration is performed only when PyTorch's version is < `min_version`
+    IMPORTANT: The symbolic must have been manually registered by the caller, otherwise
+               the incorrect symbolic may be unregistered instead.
+    """
+
+    # TODO: _unregister_custom_op_symbolic is introduced PyTorch>=1.10
+    #       Remove after PyTorch 1.10+ is used by ALL detectron2's CI
+    try:
+        from torch.onnx import unregister_custom_op_symbolic as _unregister_custom_op_symbolic
+    except ImportError:
+
+        def _unregister_custom_op_symbolic(symbolic_name, opset_version):
+            import torch.onnx.symbolic_registry as sym_registry
+            from torch.onnx.symbolic_helper import _onnx_main_opset, _onnx_stable_opsets
+
+            def _get_ns_op_name_from_custom_op(symbolic_name):
+                try:
+                    from torch.onnx.utils import get_ns_op_name_from_custom_op
+
+                    ns, op_name = get_ns_op_name_from_custom_op(symbolic_name)
+                except ImportError:
+                    if not bool(
+                        re.match(r"^[a-zA-Z0-9-_]*::[a-zA-Z-_]+[a-zA-Z0-9-_]*$", symbolic_name)
+                    ):
+                        raise ValueError(
+                            f"Invalid symbolic name {symbolic_name}. Must be `domain::name`"
+                        )
```

Comment:
Please name the exception on L220, e.g. `import_error` and add the `from import_error` for re-raising
(See https://docs.python.org/3/tutorial/errors.html#exception-chaining)

```suggestion
                        ) from import_error
```